script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -88,7 +88,7 @@ I feel like―――
 [sha:95f5b440dc0e01af4749521cb9ea2d49cd9c58e0]{
 -- Page 43, Offset 30612. Mods: @n.
 -- <昏睡|こんすい>の際で
-I saw a moon like this
+I saw a moon like this,
```

Comment:
don't think comma is needed

Reply:
There's a pause after "this" so I think it's needed

Reply:
This is all `@n` + glue, IIRC it appears as one line without much delay

Reply:
I guess I'll keep it